backend/src/models/cycle.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+/**
+ * Holds information about a recruitment cycle; attributes are subject to change.
+ *
+ * term: (for example) SP19, FA20
+ * chapter: (or school) UIUC, GaTech
```

Comment:
hmmm, I don't think we should be using abbreviations as chapters. One of my biggest annoyances looking at past spreadsheets and doing recruiting for other RSOs is that there is no standardization of naming (ie someone will write CompE and someone else will write CE and someone else will write Computer Engineering). Maybe change this comment to say `Full name of chapter (or school)` and write out `Georgia Tech not GaTech`

Reply:
agreed